### PR TITLE
[TAS-458] ✨ Add more NFT Books page

### PR DESCRIPTION
**Pull request by nwingt**
image

localhost_3000_zh-Hant_store_books (1)

image

### CODE REVIEW

File: src/mixins/nft.js [hunk 1]
```diff
@@ -184,6 +184,9 @@ export default {
         this.NFTClassMetadata.account_owner
       );
     },
+    iscnAuthor() {
```
Comment (williamchong) [inline src/mixins/nft.js:187]: this name might cause potential confusion in the future but i have no good suggestion...
`iscnMetadataAuthor` ?

File: src/pages/store/books.vue [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        href: `/api/nft/metadata?class_id=${classId}`,
+      });
+    });
+    return {
+      link,
+    };
+  },
+  computed: {
+    nftBooks() {
+      return LIKECOIN_NFT_BOOK_ITEMS.map(classId => ({ classId }));
```
Comment (williamchong) [inline src/pages/store/books.vue:52]: Should this `books` page include featured book too?

File: src/components/NFTBook/ItemCard.vue [hunk 3]
```diff
@@ -44,7 +44,7 @@
     <Label
       class="text-medium-gray mt-[6px] mb-[12px]"
       preset="p6"
-      :text="creatorDisplayName | ellipsis"
+      :text="(iscnAuthor || creatorDisplayName) | ellipsis"
```
Comment (AuroraHuang22) [inline src/components/NFTBook/ItemCard.vue:47]: I think it can be a computed property

File: src/mixins/nft.js (hunk 1 above)
Comment (nwingt) [reply src/mixins/nft.js:187]: How about iscnWorkAuthor that imply Author under CreativeWork in schema.org

File: src/components/NFTBook/ItemCard.vue (hunk 3 above)
Comment (nwingt) [reply src/components/NFTBook/ItemCard.vue:47]: can change to computed property if we used it more than once

File: src/mixins/nft.js (hunk 1 above)
Comment (williamchong) [reply src/mixins/nft.js:187]: makes sense

File: src/pages/store/books.vue (hunk 2 above)
Comment (nwingt) [reply src/pages/store/books.vue]: Yes, included

File: src/mixins/nft.js (hunk 1 above)
Comment (nwingt) [reply src/mixins/nft.js]: Updated

**Review by williamchong — APPROVED** [review]

**Pull request merged**